packages/account-balances/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@proximaone/stream-schema-accounts",
```

Comment:
there is a convention for local references https://github.com/proxima-one/stream-schemas/blob/master/tsconfig.json#L21 so either folder should be named `accounts` or package `*-account-balances`